jobs/build/ose/scripts/merge-and-build.sh

```diff
@@ -262,13 +280,27 @@ echo
 echo -n "https://brewweb.engineering.redhat.com/brew/taskinfo?taskID=${TASK_NUMBER}" > "${RESULTS}/ose-brew.url"
 brew watch-task ${TASK_NUMBER}
 
+echo
+echo "=========="
+echo "Setup: openshift-jenkins"
+echo "=========="
+rm -rf jenkins
+git clone git@github.com:openshift/jenkins.git
+OPENSHIFT_JENKINS_DIR="${WORKPATH}/jenkins/"
+cd jenkins/
+if [ "${OSE_VERSION}" != "${OSE_MASTER}" ] ; then
+  if [ "${MAJOR}" -eq 3 -a "${MINOR}" -ge 6 ] ; then # 3.5 and below maps to "release-1.5"
+    git checkout -q openshift-${MAJOR}.${MINOR}
+  fi
+fi
 
 echo
 echo "=========="
 echo "Setup: openshift-ansible"
 echo "=========="
 rm -rf openshift-ansible
```

Comment:
Looks like a pre-existing bug. The cwd is actually 'ose'. We should cd to WORKPATH before rm/clone.